src/resources/2017/Horsery.ts

```diff
@@ -0,0 +1,45 @@
+import { cliExecute, visitUrl } from "kolmafia";
+import { Modifiers } from "../..";
+import { get } from "../../property";
+
+/**
+ * @returns Whether or not the horsery is available
+ */
+export function have(): boolean {
+  return get("horseryAvailable");
+}
+
+export type Horse = "pale" | "dark" | "normal" | "crazy";
+
+/**
+ * @returns Your current horse; `null` if you are horseless
+ */
+export function current(): Horse | null {
+  return (get("_horsery") || null) as Horse | null;
+}
+
+/**
+ * @param horse The horse to change to
+ * @returns Whether, at the end of all things, that is your horse
+ */
+export function changeHorse(horse: Horse): boolean {
+  cliExecute(`horsery ${horse}`);
```

Comment:
could check current() first and not run the command at all, and should probably check have() as well

Reply:
good shout